lnbits/static/js/base.js

```diff
@@ -422,6 +422,18 @@ window.LNbits = {
       converter.setFlavor('github')
       converter.setOption('simpleLineBreaks', true)
       return converter.makeHtml(text)
+    },
+    hexToRgb: function (hex) {
```

Comment:
i dont think wrapping this into `LNbits.utils` makes sense, just use it directly

Reply:
Thought this way it would be useful for others